bridges/snowbridge/pallets/inbound-queue/src/lib.rs

```diff
@@ -140,6 +138,8 @@ pub mod pallet {
 
 		/// To withdraw and deposit an asset.
 		type AssetTransactor: TransactAsset;
+
+		type MessageProcessor: MessageProcessor;
```

Comment:
Maybe it's worth adding a short comment here explaining the purpose of the custom processor.

Reply:
*fixed*